kubernetes/start

```diff
@@ -29,7 +30,7 @@ while (( $interactive_mode == 1)) ; do
   esac
 done
 
-./cleanup
+./cleanup || true
```

Comment:
If the cleanup script fails because there's nothing to remove then we can use the `for` instruction proposed by @PI-Victor.